app/models/appeal.rb

```diff
@@ -355,7 +355,7 @@ def appellant_substitution?
   end
 
   # Determine if we are on a separate substitution appeal (used in serializer)
-  def substitution_appeal?
+  def separate_appeal_substitution?
```

Comment:
This is clearer to me and wouldn't require a code comment, assuming this is the intent:
```suggestion
  def target_of_appellant_substitution?
```

Reply:
Eh, I definitely want to hear from @n1zyy since he made this method, but I don’t think that `target_of_appellant_substitution?` is clearer.  

The method is used to determine if the appeal is a separate appeal stream, like the kind created during a pending appeal substitution.  `target_of_appellant_substitution?` sounds like it would return true for both a separate appeal stream created during a pending appeal substitution and for death dismissal substitution appeals.